airflow-core/src/airflow/ui/src/pages/Dag/Header.tsx

```diff
@@ -108,7 +111,34 @@ export const Header = ({
               />
             )}
             <FavoriteDagButton dagId={dag.dag_id} withText={true} />
-            <ParseDag dagId={dag.dag_id} fileToken={dag.file_token} />
+            <Menu.Root>
+              <Menu.Trigger asChild>
+                <Button aria-label={translate("dag:header.buttons.advanced")} variant="outline">
+                  <LuMenu />
+                </Button>
+              </Menu.Trigger>
+              <Portal>
+                <Menu.Positioner>
+                  <Menu.Content>
+                    <Menu.Item value="reparse">
+                      <div style={{ width: "100%" }}>
+                        <ParseDag dagId={dag.dag_id} fileToken={dag.file_token} width="100%" />
+                      </div>
+                    </Menu.Item>
+                    <Menu.Item closeOnSelect={false} value="delete">
+                      <div style={{ width: "100%" }}>
+                        <DeleteDagButton
+                          dagDisplayName={dag.dag_display_name}
+                          dagId={dag.dag_id}
+                          variant="outline"
```

Comment:
You don't need that:
```suggestion
```

Just delete it and the prop from the component `withText` will be ghost in short mode an outline with text.

<img width="1861" height="195" alt="Screenshot 2025-07-18 at 18 30 20" src="https://github.com/user-attachments/assets/a6a87d91-326f-42d3-aa35-879696121353" />
<img width="1893" height="562" alt="Screenshot 2025-07-18 at 18 30 30" src="https://github.com/user-attachments/assets/5cf272a9-45c5-467a-9317-f1b0f8e4a40e" />